web/src/tableUtils.ts

```diff
@@ -18,3 +18,7 @@ export function formatNumber(
     return number.toPrecision(precision).toString()
   }
 }
+
+export function getCorrectUnit(phase: string): string {
+  return phase === 'Vapor' ? ' mg/Sm3' : ' mg/Kg'
```

Comment:
mg is milligrams in the SI system. It should be micrograms, i.e. μg (or "microgram").

Reply:
ups, i'll fix